File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteFlagLinkExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteFlagLinkExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new FlagLinksBetaApi(config).DeleteFlagLink(
                projectKey: "projectKey_string",
                featureFlagKey: "featureFlagKey_string",
                id: "id_string"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FlagLinksBetaApi#DeleteFlagLink: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/DeleteFlagLinkExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.FlagLinksBetaApi(configuration).deleteFlagLink(
  "projectKey_string", // projectKey
  "featureFlagKey_string", // featureFlagKey
  "id_string", // id
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling FlagLinksBetaApi#deleteFlagLink:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
